ui/app/components/ui/eth-balance/eth-balance.component.js

```diff
@@ -0,0 +1,129 @@
+import PropTypes from 'prop-types'
+import React, {Component} from 'react'
+
+const {
+  formatBalance,
+  generateBalanceObject,
+} = require('../../../helpers/utils/util')
+const Tooltip = require('../tooltip.js')
+const FiatValue = require('../fiat-value.js')
+
+export default class EthBalance extends Component {
+  static defaultProps = {
+    style: null,
+    styleOverride: {},
+    showFiat: true,
+    needsParse: true,
+    width: undefined,
+    shorten: false,
+    incoming: false,
+  }
+
+  static propTypes = {
+    conversionRate: PropTypes.any.isRequired,
+    shorten: PropTypes.bool,
+    incoming: PropTypes.bool,
+    currentCurrency: PropTypes.any.isRequired,
```

Comment:
```suggestion
    currentCurrency: PropTypes.string.isRequired,
```
`currentCurrency` is a string. It's the currency code (e.g. `USD`)